chassis/core/src/main/java/com/griddynamics/jagger/util/generators/TestGenerator.java

```diff
@@ -23,10 +23,12 @@ static WorkloadTask generateFromTest(JLoadTest jLoadTest,
         WorkloadTask task = generatePrototype(jLoadTest.getTestDescription());
         task.setName(jLoadTest.getId());
         task.setVersion("0");
-        TerminateStrategyConfiguration terminateStrategyConfiguration = generateTermination(jLoadTest.getTermination());
-        task.setTerminateStrategyConfiguration(terminateStrategyConfiguration);
-        WorkloadClockConfiguration workloadClockConfiguration = generateLoad(jLoadTest.getLoad());
-        task.setClockConfiguration(workloadClockConfiguration);
+        task.setTestListeners(jLoadTest.getTestListeners());
```

Comment:
add standard CollectThreadsTestListener to the list as well, you have removed it in the generatePrototype()